generators/app/templates/src/components/AuthNavBar/auth-nav-bar.container.js

```diff
@@ -1,17 +1,19 @@
 import React, { Component } from "react";
 import { withWebId } from "@inrupt/solid-react-components";
+import { withTranslation } from "react-i18next";
 import AuthNavBar from "./auth-nav-bar.component";
 import data from "@solid/query-ldflex";
 
 // hasPhoto context
 const hasPhotoContext = "http://www.w3.org/2006/vcard/ns#hasPhoto";
-// img context 
-const imgContext = "http://xmlns.com/foaf/0.1/img"
+// img context
+const imgContext = "http://xmlns.com/foaf/0.1/img";
```

Comment:
are you using imageConext ? if you are using we can access by data.user.foaf_image